docs/conf.py

```diff
@@ -138,11 +137,12 @@
 # Add any paths that contain custom themes here, relative to this directory.
 # To use a different custom theme, add the directory containing the theme.
 #html_theme_path = []
-
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes. To override the custom theme, set this to the
 # name of a builtin theme or the name of a custom theme in html_theme_path.
-#html_theme = None
+#html_theme = []
```

Comment:
Can you change this back? (since the change is not needed)

Reply:
Sure, done